stubs/ink/index.d.ts

```diff
@@ -2,27 +2,34 @@ declare module 'ink' {
     import * as React from 'react';
 
     export interface RenderOptions {
-        stdout: NodeJS.WriteStream,
-        stdin: NodeJS.ReadStream,
-        debug: boolean
+        exitOnCtrlC: boolean;
+        stdout: NodeJS.WriteStream;
+        stdin: NodeJS.ReadStream;
+        debug: boolean;
     }
 
     export type Unmount = () => void;
 
     // export function renderToString<P>(tree: React.ReactElement<P>): string;
-    export function render<P>(tree: React.ReactElement<P>, options?: NodeJS.WriteStream | Partial<RenderOptions>): Unmount;
-
+    export function render<P>(
+        tree: React.ReactElement<P>,
+        options?: NodeJS.WriteStream | Partial<RenderOptions>,
+    ): {
+        rerender: <P>(tree: React.ReactElement<P>) => void;
+        unmount: Unmount;
+        waitUntilExit: Promise<void>;
+    };
 
     export interface ColorProps {
```

Comment:
would be great to pull these out of `chalk`, but I don't know if that's possible?

https://github.com/chalk/chalk/blob/60959e05cfe7086a38927eae5b5e04c459ce4182/index.d.ts#L93-L265